HW6/task3.py

```diff
@@ -0,0 +1,5 @@
+lst = [1, 2, "qwerty", "asd", "zxc", 4, 6, 7]
+
+
+filtered_list = list(filter(lambda x: x == int, lst))
```

Comment:
almost there, but the type of `x` should be compared to `int` or `isinstance` function can be used here